go/chat/utils/utils.go

```diff
@@ -916,7 +918,12 @@ func GetDesktopNotificationSnippet(conv *chat1.ConversationLocal, currentUsernam
 	if !mvalid.IsEphemeral() {
 		switch msg.GetMessageType() {
 		case chat1.MessageType_REACTION:
-			snippet = "reacted to your message."
+			reaction, err := GetReaction(msg)
```

Comment:
should we do a similar check for TEXT to make sure the body type matches the header type? otherwise we can ditch that method to just do `Valid().MessageBody().Reaction()` directly

Reply:
I guess.